tests/foreman/ui/test_partitiontable.py

```diff
@@ -89,6 +90,36 @@ def test_positive_create_with_name(self):
 
     @run_only_on('sat')
     @tier1
+    def test_positive_view_with_html_name(self):
+        """Create a new partition table that has html tags in its name. Then
+        open it for preview
+
+        :id: 2d0c8bff-52fc-43a8-9820-147b125c1bc9
+
+        :expectedresults: Partition table is opened successfully and has
+            correct name
+
+        :BZ: 1225857
+
+        :CaseImportance: Critical
```

Comment:
Medium

Reply:
I would say that all BZs covered with customer tickets attached should be treated as critical functionality, but let me know if that statement is wrong

Reply:
oh come on!

Reply:
Oh come on! ` I raise a ticket that the icon is not showing right` - then it will not have urgent/high severity, right?
I don't see any proper arguments from your side, so I am going to follow my current strategy